llama_stack/providers/impls/meta_reference/scoring/scoring.py

```diff
@@ -13,23 +13,20 @@
 from llama_stack.apis.datasets import *  # noqa: F403
 from llama_stack.apis.inference.inference import Inference
 from llama_stack.providers.datatypes import ScoringFunctionsProtocolPrivate
-from llama_stack.providers.impls.meta_reference.scoring.scoring_fn.equality_scoring_fn import (
-    EqualityScoringFn,
-)
-
-from llama_stack.providers.impls.meta_reference.scoring.scoring_fn.llm_as_judge_scoring_fn import (
-    LlmAsJudgeScoringFn,
-)
-
-from llama_stack.providers.impls.meta_reference.scoring.scoring_fn.subset_of_scoring_fn import (
-    SubsetOfScoringFn,
-)
 
 from .config import MetaReferenceScoringConfig
+from .scoring_fn.answer_parsing_scoring_fn import AnswerParsingScoringFn
+from .scoring_fn.equality_scoring_fn import EqualityScoringFn
+from .scoring_fn.llm_as_judge_scoring_fn import LlmAsJudgeScoringFn
+from .scoring_fn.subset_of_scoring_fn import SubsetOfScoringFn
 
 FIXED_FNS = [EqualityScoringFn, SubsetOfScoringFn]
 
-LLM_JUDGE_FNS = [LlmAsJudgeScoringFn]
+# Scoring functions with context that can be registered
+REGISTERABLE_SCORING_FNS = {
```

Comment:
Each Registerable ScoringFn is mapped to a ScoringContextType, E.g. So that we are able to register a scoring function with custom judge_prompt/answer_extraction regex.